LoRaEngine/test/LoRaWanNetworkServer.Test/MultiGatewayFrameCounterUpdateStrategyTest.cs

```diff
@@ -45,17 +45,17 @@ public async Task When_Device_Has_No_Changes_To_Fcnt_Should_Not_Save_Changes(int
         }
 
         [Theory]
-        [InlineData(0)]
-        [InlineData(9)]
-        public async Task When_Device_Has_Up_To_9_Changes_In_Fcnt_Up_Should_Not_Save_Changes(int startFcntUp)
+        // [InlineData(0U)]
```

Comment:
Is the test failing with initial fcntup 0?

Reply:
It was in the beginning. All good now. Enabled again.